force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -62,14 +69,30 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<Boolean> checkStringContent(String input) {
         // Initialize a list to store the results
+        List<Boolean> results = new List<Boolean>();       
         
         // Check if the string starts with 'Sales'
+        If(input.startsWith('Sales')){            
+          results.add(True);
+        }else {
+            results.add(False);
+        }
```

Comment:
So, for each of these, the `input.startsWith()` call returns a boolean. You don't need to check the result of a boolean to get a boolean.
```suggestion
        results.add(input.startsWith('Sales'));           
```

Reply:
Thank you. I will work on this.